integration/spark2/src/main/scala/org/apache/spark/sql/CarbonDataFrameWriter.scala

```diff
@@ -81,6 +81,7 @@ class CarbonDataFrameWriter(sqlContext: SQLContext, val dataFrame: DataFrame) {
   private def makeCreateTableString(schema: StructType, options: CarbonOption): String = {
     val property = Map(
       "SORT_COLUMNS" -> options.sortColumns,
+      "SORT_SCOPE" -> options.sortScope,
```

Comment:
From this PR, it seems that carbon will not output error message even if there is some table properties it can not recognized during create table by DataFrame API. I think it is better to add a check for it.

Reply:
I can't list all of the properties of table, any class record this ?

Reply:
I think you can refer to CREATE TABLE section in http://carbondata.apache.org/data-management-on-carbondata.html